CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## 1.2.1
```

Comment:
Nit-pick: per SemVer a non-breaking *enhancement* is a minor version bump (so we'd be at 1.3.0).

Reply:
Good point, changed to 1.3.0